MetaMask/metamask-extension
# fix: update backup copy

## **Description**



- Update the copy in the "Advanced Settings" to reflect the proper action.
- Update all the mentions of _backup_ (noun) to _back up_ (verb)

## **Related issues**

Fixes: https://github.com/MetaMask/accounts-planning/issues/141

## **Manual testing steps**

1. Go to Settings > Advanced
2. Check that the copy for the backup options are updated

## **Screenshots/Recordings**



### **Before**

<img width="349" alt="Screenshot 2023-11-15 at 10 17 59" src="https://github.com/MetaMask/metamask-extension/assets/17601467/15f5932e-881c-404a-bc19-a688cc3d04cc">

### **After**

<img width="349" alt="Screenshot 2023-11-15 at 10 17 59" src="https://github.com/MetaMask/metamask-extension/assets/17601467/c2b7988d-ef4e-4102-a229-7a7b36345292">

## **Pre-merge author checklist**

  - In case it's not yet "ready for review", I've set it to "draft".
  - In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**
